create-pull-request.py

```diff
@@ -39,8 +42,12 @@ def pr_branch_exists(repo, branch):
 
 
 def get_head_author(github_event):
-    email = "{head_commit[author][email]}".format(**github_event)
-    name = "{head_commit[author][name]}".format(**github_event)
+    if 'schedule' in github_event:
+        email=os.environ['GITHUB_ACTOR']
+        name=os.environ['GITHUB_ACTOR'] + '@users.noreply.github.com'
```

Comment:
In case of a scheduled workflow this variable was set to my user, but I have no idea why. I expected to see some GitHub Bot there. Anyway, let me know what do you think

Reply:
It is a bit odd that GitHub executes scheduled events with a user account and not a bot user, I agree. Perhaps it's to help with cases just like this where a valid user is needed to perform further action. I don't know how GitHub determines which user account to use and the [documentation](https://help.github.com/en/articles/virtual-environments-for-github-actions#default-environment-variables) doesn't mention. I'm going to guess that it's the user account of the HEAD commit. I think using `GITHUB_ACTOR` in the way you have done here is fine for now. If there happen to be cases when it's not set to a valid user then we might need to explicitly set a user account via environment variables on the action itself.